src/storage.rs

```diff
@@ -68,26 +68,49 @@ impl Default for AccountBasic {
     }
 }
 
-/// EVM Code, currently mapping to REVM's [ByteCode::LegacyAnalyzed].
-// TODO: Support raw legacy & EOF
 #[derive(Debug, Clone, PartialEq, Default, Serialize, Deserialize)]
-pub struct EvmCode {
-    /// Bytecode with 32 zero bytes padding
+pub struct LegacyAnalyzedCode {
```

Comment:
We can just call this `LegacyCode` that we enforce to be analyzed for performance.